src/js/blocksetting123.js

```diff
@@ -180,13 +176,13 @@ function practice_block1(timeline, jsPsych) {
       c1++;
     }
     if (counterP_1 === 1) {
-      x1 = nums2_1[n];
+      x1 = nums2_1[n]; // this generates the random number 1 -359
     }
     if (counterP_1 !== 1) {
       // x1 = x1
     }
     // make task slightly easier for practicing with lower noise stdev -- CHANGED SO NOT TRUE (REALISTIC TO TASK)
-    outcome = Math.mod(normalRandomScaled(x1, 20), 360);
+    outcome = Math.mod(normalRandomScaled(x1, 25), 360); //x1 = mean, 20 = stdev; 360 = 
```

Comment:
Weren't we just going to increase the shield size first, and then try also increasing the noise?